momentum/4/config/ref-rollup-mx-name.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "rollup_mx_name"
+description: "If the associated rollup_mx option is set to 'true', rollup_mx_name specifies the MX host domain to route the traffic to. Valid in host, binding, and binding_group scopes. Usable in host stanzas, which can be included in binding or binding group scopes."
```

Comment:
This is misleading: the rollup_mx_name must be in a Host stanza, which itself can be global, inside a binding, or inside a binding_group stanza.

Reply:
Text adjusted.